go/p2p/host.go

```diff
@@ -197,3 +206,28 @@ func (cfg *ConnGaterConfig) Load() error {
 
 	return nil
 }
+
+// NewConnManager constructs a new connection manager.
+func NewResourceManager() (network.ResourceManager, error) {
+	// Use the default resource manager for non-seed nodes.
+	if config.GlobalConfig.Mode != config.ModeSeed {
+		return nil, nil
+	}
+
+	// Tweak limits for seed nodes.
+	defaultLimits := rcmgr.DefaultLimits
+	defaultLimits.SystemBaseLimit.ConnsInbound = rcmgr.DefaultLimits.SystemBaseLimit.ConnsOutbound
```

Comment:
Maybe add a comment about what these values are (e.g. 64 -> 128). Imo, it might actually be nicer to specify values here (with some comments) instead of reusing variables from the default limits.

Btw do we expect these to be enough? On mainnet there's a lot more nodes (vs testnet) that will be trying to connect to the seed node. I guess it should be fine assuming the connections will get closed quickly and clients will be retrying.

Reply:
Yes, they should be enough. As stated in the description, the connection manager will trim connections if resource manager's limits are larger than the high watermark.